File: src/main/java/me/devsnox/playtime/varoxtime/TimePlayer.java
package me.devsnox.playtime.varoxtime;

import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

/**
 * Copyright by DevSnox
 * E-Mail: me.devsnox@gmail.com
 * Skype: DevSnox
 */
public class TimePlayer {

    @Getter @Setter
    private UUID uuid;

    @Getter @Setter
    private long time;

    public TimePlayer(UUID uuid, long time) {
        this.uuid = uuid;
        this.time = time;
    }
}

File: src/main/java/me/devsnox/playtime/playtime/TimePlayer.kt
package me.devsnox.playtime.playtime;

import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

/**
 * Created by Yasin Dalal
 * Copyright (c) 2017-2019 Yasin Dalal
 * GitHub: https://github.com/DevSnox
 * E-Mail: yasin@dalal.ch
 */
public class TimePlayer {

    @Getter @Setter
    private UUID uuid;

    @Getter @Setter
    private long time;

    public TimePlayer(UUID uuid, long time) {
        this.uuid = uuid;
        this.time = time;
    }
}
